released/packages/coq-geocoq/coq-geocoq.2.3.0/opam

```diff
@@ -17,7 +17,7 @@ install: [
 ]
 remove: ["rm" "-R" "%{lib}%/coq/user-contrib/GeoCoq"]
 depends: [
-  "coq" { (>= "8.5" & < "8.6~") | (>= "8.6.1" & < "8.7~") | (>= "8.7" & < "8.8~") }
+  "coq" { (>= "8.5" & < "8.6~") | (>= "8.6.1" & < "8.7~") | (>= "8.7" & < "8.8~") | (>= "8.8" & < "8.9~") }
```

Comment:
Would you mind changing this to `(>= "8.5 & < "8.9~")` ? Just nitpicking, but I guess you see my point...

Reply:
Ok but if I remember well,  8.6 should be excluded.